### PR TITLE
PI is viable environment for running tests

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: spec/environment/pi.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module Environment
+    module ConnectionHelper
```
Comment (yenkins) [inline spec/environment/pi.rb]: Has no descriptive comment (IrresponsibleModule)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/128/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/956/ : SUCCESS in 52s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1225/ : SUCCESS in 40s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/45/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/964/ : SUCCESS in 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1233/ : SUCCESS in 55s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1389/ : SUCCESS in 59m 40s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1154/ : UNSTABLE in 3m 30s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/61/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/983/ : SUCCESS in 45s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1252/ : SUCCESS in 51s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1405/ : SUCCESS in 56m 06s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1170/ : UNSTABLE in 16m 49s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/62/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/993/ : SUCCESS in 2m 11s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1262/ : SUCCESS in 2m 07s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1412/ : SUCCESS in 54m 40s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1179/ : FAILURE in 32s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/65/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/998/ : SUCCESS in 46s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1267/ : SUCCESS in 2m 25s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1416/ : ABORTED in 2h 00m 03s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-lcm-integration-tests-zuul-el7/1/ : FAILURE in 1m 01s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/233/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1075/ : SUCCESS in 2m 44s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1344/ : SUCCESS in 2m 32s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/80/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/1077/ : SUCCESS in 3m 01s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1346/ : SUCCESS in 2m 39s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1441/ : SUCCESS in 18m 45s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1193/ : SUCCESS in 34m 40s

**Pull request merged**